python/tk_desktop2/websockets/shotgun_cert_handler.py

```diff
@@ -54,7 +55,8 @@ def _retrieve_certificates_from_shotgun(self):
         """
         logger.debug("Retrieving certificates from Shotgun")
         certs = self._bundle.shotgun._call_rpc("sg_desktop_certificates", {})
-        self._write_cert(self.cert_path, certs["sg_desktop_cert"])
+        self._write_cert(
```

Comment:
Seems legit to me!
Funny, we have the [roughly the same line](https://github.com/shotgunsoftware/tk-framework-desktopserver/blob/6d1b91f853cac2311806f1b54998aa05e1f71628/framework.py#L257) on our end

Reply:
Our implementation is a tad more complicated because OpenSSL won't just accept the entire chain and we have to call a different method to initialize the SSL context.